controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -43,8 +46,9 @@ import (
 var operatorNamespace string
 
 const (
-	wrongNamespacedName              = "Ignoring this resource. Only one should exist, and this one has the wrong name and/or namespace."
-	random30CharacterString          = "KP7TThmSTZegSGmHuPKLnSaaAHSG3RSgqw6akBj0oVk"
+	wrongNamespacedName     = "Ignoring this resource. Only one should exist, and this one has the wrong name and/or namespace."
```

Comment:
I can still see the word "wrong" in the message even after your update

Reply:
Since this was only used once, we probably didn't need a const with the error message. Modified it inline and modified it to show what's expected and what's received by the reconciler